src/main/kotlin/no/nav/syfo/service/OppfolgingsplanVarselService.kt

```diff
@@ -55,6 +55,36 @@ class OppfolgingsplanVarselService(
         )
     }
 
+    suspend fun sendOppfolgingsplanForesporselVarselTilNarmesteLeder(
+        varselHendelse: NarmesteLederHendelse
+    ){
+        senderFacade.sendTilDineSykmeldte(
+            varselHendelse,
+            DineSykmeldteVarsel(
+                ansattFnr = varselHendelse.arbeidstakerFnr,
+                orgnr = varselHendelse.orgnummer,
+                oppgavetype = varselHendelse.type.toDineSykmeldteHendelseType().toString(),
+                lenke = null,
+                tekst = "DINE_SYKMELDTE_OPPFOLGINGSPLAN_FORESPORSEL_TEKST",
+                utlopstidspunkt = OffsetDateTime.now().plusWeeks(WEEKS_BEFORE_DELETE)
+            )
+        )
+        senderFacade.sendTilArbeidsgiverNotifikasjon(
+            varselHendelse,
+            ArbeidsgiverNotifikasjonInput(
+                UUID.randomUUID(),
+                varselHendelse.orgnummer,
+                varselHendelse.narmesteLederFnr,
+                varselHendelse.arbeidstakerFnr,
+                ARBEIDSGIVERNOTIFIKASJON_OPPFOLGING_MERKELAPP,
+                "ARBEIDSGIVERNOTIFIKASJON_OPPFOLGINGSPLAN_FORESPORSEL_MESSAGE_TEXT",
```

Comment:
Kan det være at meningen til Luan er at disse konstantene skulle få en verdi definert et sted?

Reply:
Litt usikker, men forsto fra Luan sin [kommentar](https://nav-it.slack.com/archives/CGX6MF9S8/p1736335746403879?thread_ts=1736335278.599559&cid=CGX6MF9S8) at vi tar kun nais-config i den PRet, så dere kan komme dere videre. 
Så løser vi konstantene og andre ting senere. 
Stemmer det @geir-waagboe ?

Reply:
Ja, stemmer det dere tenkte.

@geir-waagboe Ja, det er placeholder for konstanter som Stine jobber, de blir definert et annet sted når tekstene er klare.